src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,72 @@
+package seedu.addressbook.common;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.commands.*;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.data.person.*;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.List;
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+
+public class UtilsTest {
+
+    @Test
+    public void isAnyNull_returnsFalse() throws IllegalValueException {
```

Comment:
Remember that the naming convention is TestName_Scenario_Result, or just TestName if you want to widen the scope of the test.